aconfigtoml/go.mod

```diff
@@ -1,8 +1,10 @@
 module github.com/cristalhq/aconfig/aconfigtoml
 
-go 1.16
+go 1.21.0
```

Comment:
Do we need this bump?

Reply:
It's generated automatically.
The [pelletier/go-toml](https://github.com/pelletier/go-toml/blob/af236b689f5ac1bdbdfeb8af1ed7b398a2dbe05e/go.mod#L3) declare the golang version

Reply:
If you know the underlying reason of [BurntSushi/toml](https://github.com/BurntSushi/toml), we won't need to exchange it.